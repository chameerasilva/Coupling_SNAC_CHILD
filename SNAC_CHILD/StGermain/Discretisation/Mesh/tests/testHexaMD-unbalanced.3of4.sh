#!/bin/sh

TEST_SCRIPT=./VMake/executableTester.sh
until test -r ${TEST_SCRIPT} ; do
        TEST_SCRIPT=../${TEST_SCRIPT}
done
. ${TEST_SCRIPT}

runAndHandleSystemTest "testHexaMD --allowUnusedCPUs=False --allowUnbalancing=True --meshSizeI=11 --meshSizeJ=2 --meshSizeK=2" "$0" "$@"
