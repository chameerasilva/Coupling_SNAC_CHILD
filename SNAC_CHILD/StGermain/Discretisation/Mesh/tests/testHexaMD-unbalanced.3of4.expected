StGermain Framework revision 3570. Copyright (C) 2003-2005 VPAC.
Corner Node Layout
Current processor is 3 of 4 in use (4 total).
allowUnusedCPUs: False
allowPartitionOnNode: True
allowPartitionOnElement: False
allowUnbalancing: True
shadowDepth: 1
procTopology:
	IJKTopology (ptr): (0x8060da8)
		size : (4,1,1)
		isPeriodic : (False,False,False)
		dynamicSizes : True
partitionedAxis: { True, False, False }
partitionCounts: { 4, 1, 1 }
elementGlobalCounts: { 10, 1, 1 }
elementGlobalCount: 10
	elementLocalCounts[0]: { 3, 1, 1 }
	elementLocalCounts[1]: { 3, 1, 1 }
	elementLocalCounts[2]: { 2, 1, 1 }
	elementLocalCounts[3]: { 2, 1, 1 }
elementDomainCounts: { 3, 1, 1 }
nodeGlobalCounts: { 11, 2, 2 }
nodeGlobalCount: 44
	nodeLocalCounts[0]: { 4, 2, 2 }
	nodeLocalCounts[1]: { 4, 2, 2 }
	nodeLocalCounts[2]: { 3, 2, 2 }
	nodeLocalCounts[3]: { 3, 2, 2 }
Element, global: 0 - local: X - shadow: X - domain: X
Element, global: 1 - local: X - shadow: X - domain: X
Element, global: 2 - local: X - shadow: X - domain: X
Element, global: 3 - local: X - shadow: X - domain: X
Element, global: 4 - local: X - shadow: X - domain: X
Element, global: 5 - local: X - shadow: X - domain: X
Element, global: 6 - local: X - shadow: X - domain: X
Element, global: 7 - local: X - shadow: 0 - domain: 2
Element, global: 8 - local: 0 - shadow: X - domain: 0
Element, global: 9 - local: 1 - shadow: X - domain: 1

Element, local: 0 - global: 8
Element, local: 1 - global: 9

Element, domain: 0 - global: 8
Element, domain: 1 - global: 9
Element, domain: 2 - global: 7

Node, global: 0 - local: X - shadow: X - domain: X
Node, global: 1 - local: X - shadow: X - domain: X
Node, global: 2 - local: X - shadow: X - domain: X
Node, global: 3 - local: X - shadow: X - domain: X
Node, global: 4 - local: X - shadow: X - domain: X
Node, global: 5 - local: X - shadow: X - domain: X
Node, global: 6 - local: X - shadow: X - domain: X
Node, global: 7 - local: X - shadow: 0 - domain: 12
Node, global: 8 - local: 0 - shadow: X - domain: 0
Node, global: 9 - local: 1 - shadow: X - domain: 1
Node, global: 10 - local: 2 - shadow: X - domain: 2
Node, global: 11 - local: X - shadow: X - domain: X
Node, global: 12 - local: X - shadow: X - domain: X
Node, global: 13 - local: X - shadow: X - domain: X
Node, global: 14 - local: X - shadow: X - domain: X
Node, global: 15 - local: X - shadow: X - domain: X
Node, global: 16 - local: X - shadow: X - domain: X
Node, global: 17 - local: X - shadow: X - domain: X
Node, global: 18 - local: X - shadow: 1 - domain: 13
Node, global: 19 - local: 3 - shadow: X - domain: 3
Node, global: 20 - local: 4 - shadow: X - domain: 4
Node, global: 21 - local: 5 - shadow: X - domain: 5
Node, global: 22 - local: X - shadow: X - domain: X
Node, global: 23 - local: X - shadow: X - domain: X
Node, global: 24 - local: X - shadow: X - domain: X
Node, global: 25 - local: X - shadow: X - domain: X
Node, global: 26 - local: X - shadow: X - domain: X
Node, global: 27 - local: X - shadow: X - domain: X
Node, global: 28 - local: X - shadow: X - domain: X
Node, global: 29 - local: X - shadow: 2 - domain: 14
Node, global: 30 - local: 6 - shadow: X - domain: 6
Node, global: 31 - local: 7 - shadow: X - domain: 7
Node, global: 32 - local: 8 - shadow: X - domain: 8
Node, global: 33 - local: X - shadow: X - domain: X
Node, global: 34 - local: X - shadow: X - domain: X
Node, global: 35 - local: X - shadow: X - domain: X
Node, global: 36 - local: X - shadow: X - domain: X
Node, global: 37 - local: X - shadow: X - domain: X
Node, global: 38 - local: X - shadow: X - domain: X
Node, global: 39 - local: X - shadow: X - domain: X
Node, global: 40 - local: X - shadow: 3 - domain: 15
Node, global: 41 - local: 9 - shadow: X - domain: 9
Node, global: 42 - local: 10 - shadow: X - domain: 10
Node, global: 43 - local: 11 - shadow: X - domain: 11

Node, local: 0 - global: 8
Node, local: 1 - global: 9
Node, local: 2 - global: 10
Node, local: 3 - global: 19
Node, local: 4 - global: 20
Node, local: 5 - global: 21
Node, local: 6 - global: 30
Node, local: 7 - global: 31
Node, local: 8 - global: 32
Node, local: 9 - global: 41
Node, local: 10 - global: 42
Node, local: 11 - global: 43

Node, domain: 0 - global: 8
Node, domain: 1 - global: 9
Node, domain: 2 - global: 10
Node, domain: 3 - global: 19
Node, domain: 4 - global: 20
Node, domain: 5 - global: 21
Node, domain: 6 - global: 30
Node, domain: 7 - global: 31
Node, domain: 8 - global: 32
Node, domain: 9 - global: 41
Node, domain: 10 - global: 42
Node, domain: 11 - global: 43
Node, domain: 12 - global: 7
Node, domain: 13 - global: 18
Node, domain: 14 - global: 29
Node, domain: 15 - global: 40


Body Node Layout
Checking body node decomp has same element decomp as corner node decomp:
	Checking general partitions etc
	Checking the 10 individual items
	The decomps matched for the given item types
	Passed.
Checking body node decomp has same node decomp as it's element decomp:
	Checking general partitions etc
	Checking the 10 individual items
	The decomps matched for the given item types
	Passed.
